src/App.js

```diff
@@ -10,6 +10,7 @@ import AlertUnsubscribe from "./scenes/AlertUnsubscribe";
 import ForgotPassword from "./scenes/ForgotPassword";
 import About from "./scenes/About";
 import Terms from "./scenes/Terms";
+import MeirimAdmin from "./scenes/Admin/MeirimAdmin";
```

Comment:
Remove Meirim from the naming of variables and files.
We want to leave the code generic.